--- nginx.spec.orig	2016-04-19 17:19:41.000000000 +0000
+++ nginx.spec	2016-05-10 22:00:06.121018827 +0000
@@ -62,18 +62,18 @@ BuildRequires: libGeoIP-devel
 # end of distribution specific definitions
 
 %define main_version                 1.9.15
-%define main_release                 1%{?dist}.ngx
+%define main_release                 1%{?dist}.ngx_oro
 %define module_xslt_version          %{main_version}
-%define module_xslt_release          1%{?dist}.ngx
+%define module_xslt_release          1%{?dist}.ngx_oro
 %define module_geoip_version         %{main_version}
-%define module_geoip_release         1%{?dist}.ngx
+%define module_geoip_release         1%{?dist}.ngx_oro
 %define module_image_filter_version  %{main_version}
-%define module_image_filter_release  1%{?dist}.ngx
+%define module_image_filter_release  1%{?dist}.ngx_oro
 %define module_perl_version          %{main_version}
-%define module_perl_release          1%{?dist}.ngx
+%define module_perl_release          1%{?dist}.ngx_oro
 %define module_njs_shaid             1c50334fbea6
 %define module_njs_version           0.0.20160414.%{module_njs_shaid}
-%define module_njs_release           1%{?dist}.ngx
+%define module_njs_release           1%{?dist}.ngx_oro
 
 %define bdir %{_builddir}/%{name}-%{main_version}
 
@@ -121,6 +121,8 @@ BuildRequires: libGeoIP-devel
         --with-mail_ssl_module \
         --with-file-aio \
         --with-ipv6 \
+        --add-module=./naxsi_src \
+        --add-module=./headers-more-nginx-module \
         %{?with_http2:--with-http_v2_module}")
 
 Summary: High performance web server
@@ -144,6 +146,9 @@ Source10: nginx.suse.logrotate
 Source11: nginx-debug.service
 Source12: COPYRIGHT
 Source13: njs-%{module_njs_shaid}.tar.gz
+Source14: naxsi.tar.gz
+Source15: naxsi_core.rules
+Source16: headers-more-nginx-module.tar.gz
 
 License: 2-clause BSD-like license
 
@@ -212,11 +217,13 @@ Dynamic nJScript module for nginx.
 %prep
 %setup -q
 tar xvzf %SOURCE13
+tar xvzf %SOURCE14
 cp %{SOURCE2} .
 sed -e 's|%%DEFAULTSTART%%|2 3 4 5|g' -e 's|%%DEFAULTSTOP%%|0 1 6|g' \
     -e 's|%%PROVIDES%%|nginx|g' < %{SOURCE2} > nginx.init
 sed -e 's|%%DEFAULTSTART%%||g' -e 's|%%DEFAULTSTOP%%|0 1 2 3 4 5 6|g' \
     -e 's|%%PROVIDES%%|nginx-debug|g' < %{SOURCE2} > nginx-debug.init
+tar xvzf %SOURCE16
 
 %build
 ./configure %{COMMON_CONFIGURE_ARGS} \
@@ -276,6 +283,8 @@ cd $RPM_BUILD_ROOT%{_sysconfdir}/nginx &
     $RPM_BUILD_ROOT%{_sysconfdir}/nginx/nginx.conf
 %{__install} -m 644 -p %{SOURCE5} \
     $RPM_BUILD_ROOT%{_sysconfdir}/nginx/conf.d/default.conf
+%{__install} -m 644 -p %{SOURCE15} \
+    $RPM_BUILD_ROOT%{_sysconfdir}/nginx/naxsi_core.rules
 
 %{__mkdir} -p $RPM_BUILD_ROOT%{_sysconfdir}/sysconfig
 %{__install} -m 644 -p %{SOURCE3} \
@@ -349,6 +358,7 @@ cd $RPM_BUILD_ROOT%{_sysconfdir}/nginx &
 %config(noreplace) %{_sysconfdir}/nginx/koi-utf
 %config(noreplace) %{_sysconfdir}/nginx/koi-win
 %config(noreplace) %{_sysconfdir}/nginx/win-utf
+%config(noreplace) %{_sysconfdir}/nginx/naxsi_core.rules
 
 %config(noreplace) %{_sysconfdir}/logrotate.d/nginx
 %config(noreplace) %{_sysconfdir}/sysconfig/nginx
